services/src/main/scala/cromwell/services/metadata/impl/MetadataServiceActor.scala

```diff
@@ -58,6 +58,7 @@ case class MetadataServiceActor(serviceConfig: Config, globalConfig: Config, ser
   val dbFlushRate = serviceConfig.getOrElse("db-flush-rate", 5.seconds)
   val dbBatchSize = serviceConfig.getOrElse("db-batch-size", 200)
   val writeActor = context.actorOf(WriteMetadataActor.props(dbBatchSize, dbFlushRate, serviceRegistryActor, LoadConfig.MetadataWriteThreshold), "WriteMetadataActor")
+  val deleteActor = context.actorOf(Props(new DeleteMetadataActor), "DeleteMetadataActor")
```

Comment:
The `props` method in the companion object is a pretty standard pattern in Cromwell (examples above) though I'm not 💯 why